docs/src/componentDocs/DrawerSubheader/playground/PlaygroundPage.tsx

```diff
@@ -94,7 +94,9 @@ const generateSnippet: CodeSnippetFunction = (data) =>
     ${getPropsToString(getPropsMapping(data, inputConfig), { join: '\n\t', skip: ['open'] })}
 >
     <Box sx={{ p: 2 }}>Subheader Content Here</Box>
-</DrawerSubheader>`.replace(/^\s*$(?:\r\n?|\n)/gm, '');
+</DrawerSubheader>`
+        .replace(/^\s*$(?:\r\n?|\n)/gm, '')
```

Comment:
Can we move this to shared utility? So, the other component's code snippet will be fixed.

Reply:
Yes adding to backlog but we just might update the getPropsToString